modules/ossm-migrating-hub-premigration-checklists.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/ossm-migrating-from-2-to-3-assembly.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-hub-premigration-checklists_{context}"]
+= Premigration checklists
+
+There are a series of premigration checklists that must be completed before you can begin your migration.
```

Comment:
```suggestion
Before you can begin your migration, complete the premigration checklists.
```
You can engage the reader a bit better by using the imperative mood: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#mood